packages/editor/test/Sx.tsx

```diff
@@ -9,17 +9,20 @@ import {
 import { ThemeProvider } from 'theme-ui'
 import { Sx } from '../src'
 
+const noop = () => {}
+
 afterEach(cleanup)
 
-if (global.document) {
-  document.createRange = () => ({
-    setStart: () => {},
-    setEnd: () => {},
-    commonAncestorContainer: {
-      nodeName: 'BODY',
-      ownerDocument: document,
-    },
-  })
+if ((global as any).document) {
```

Comment:
I used `any` within tests as they are not within the production code. Hope that's okay.

Reply:
yeah, don't worry about it